.github/workflows/build-test-distribute.yaml

```
@@ -75,6 +78,9 @@ jobs:
           echo "images=$(make images/info/release/json)" >> $GITHUB_OUTPUT
           echo "registry=$(make docker/info/registry)" >> $GITHUB_OUTPUT
           echo "version=$(make build/info/version)" >> $GITHUB_OUTPUT
+          echo "distribution_repository=$(make build/info/distribution/repo)" >> $GITHUB_OUTPUT
```

```suggestion
          echo "cloudsmith_repository=$(make build/info/cloudsmith_repository)" >> $GITHUB_OUTPUT
```

Let's call the things by what they are it removes future confusion...

---

done